test/gtest/common/gtest-all.cc

```diff
@@ -7489,7 +7489,7 @@ void StackLowerThanAddress(const void* ptr, bool* result) {
 }
 
 bool StackGrowsDown() {
-  int dummy;
+  int dummy = 0;
```

Comment:
why needed?

Reply:
This change was applied along with cherry pick of [UCS/SOCK: Fix RPM build with gcc11 on fedora34](https://github.com/openucx/ucx/commit/81b169f2b562c835db41f58af7016fe3beec3459)